db/dbBenchmarkTests.js

```diff
@@ -0,0 +1,98 @@
+require('dotenv').config();
+const { MongoClient } = require('mongodb');
+const Promise = require('bluebird');
+
+const dbAddress = process.env.DB_ADDRESS || 'localhost';
+const dbName = process.env.DB_NAME || 'marzagat_overview';
+const collectionName = process.env.COLLECTION_NAME || 'restaurants';
+const dbSize = parseInt(process.env.DB_SIZE, 10) || 10000000;
+const index = process.env.INDEX || '_id';
+
+const connectToDb = async () => {
+  const url = `mongodb://${dbAddress}/`;
+  const client = await MongoClient.connect(url);
+  const collection = client.db(dbName).collection(collectionName);
+  return { client, collection };
+};
+
+const timer = async (testDescription, callback) => {
+  const startTime = new Date().getTime();
+  await callback();
+  const endTime = new Date().getTime();
+  const executionTime = endTime - startTime;
+  console.log(`${testDescription}: executed in ${executionTime} ms`);
+};
+
+const getRandomId = () => Math.floor(Math.random() * dbSize);
+
+const getOptions = (id) => {
+  const options = {};
+  if (index !== '_id') {
+    options[index] = id.toString();
+  } else {
+    options[index] = id;
+  }
+
+  return options;
+};
+
+const runTests = async (name1, name2) => {
+  console.log(`\n\nTESTING SCHEMA WITH INDEX: ${index}`);
+  console.log('===================================\n');
+
+  const { client, collection } = await connectToDb();
+  let testId;
+
+  testId = getRandomId();
```

Comment:
very small, but is there a reason not to put the ``testId`` declaration and assignment on one line?

Reply:
Wondering the reason to have these parameters ``name1``, ``name2``? didn't see them used in ``runTests``

Reply:
oops good catch on `name1` and `name2` - was something i was initially thinking of using and then didn't, but forgot to take them out of the params. 

re: the testId, since I change it before a few different tests somehow I felt like I wanted it to be declared on its own so it doesn't look like it's just associated with any one test, and then each test sets its value separately. not sure if that makes sense or not, any thoughts?

Reply:
that makes sense regarding ``testId``, just curious. could also consider declaring it at the top, too. but as is works, probably just a matter of preference.